lib/models/store_product_wrapper.dart

```diff
@@ -1,16 +1,16 @@
 import 'package:freezed_annotation/freezed_annotation.dart';
 
-import 'discount.dart';
 import 'introductory_price.dart';
+import 'store_product_discount.dart';
 
-part 'product_wrapper.freezed.dart';
-part 'product_wrapper.g.dart';
```

Comment:
maybe we can get rid of the `wrapper` part now? we're definitely inconsistent about naming, with some stuff having a `wrapper` prefix and others not having it

Reply:
like, customerInfo and storeTransaction don't have the suffix

Reply:
good call. I wonder if changing the name will break people's imports? If they are not importing object_wrappers.dart it will right?